docker/compose-local-dev-traces-v2.overrides.yml

```diff
@@ -22,7 +21,7 @@ services:
   traces-node-v2:
     hostname: traces-node-v2
     container_name: traces-node-v2
-    image: consensys/linea-besu:24.12-delivery40
+    image: consensys/linea-besu-package:devnet-0d2fbde
     profiles: [ "l2", "l2-bc", "debug", "external-to-monorepo" ]
     depends_on:
       traces-node-v2-plugin-downloader:
```

Comment:
This line above and the downloda service itself can be removed, right?

Reply:
Yes, I forgot removing it here. Will do, thanks!